includes/functions/llms.functions.certificate.php

```diff
@@ -486,4 +486,13 @@ function llms_register_certificate_image_size() {
 	add_image_size( 'lifterlms_certificate_background', $width, $height, true );
 
 }
-add_action( 'after_setup_theme', 'llms_register_certificate_image_size' );
+/**
+ * Register and enqueue a local font cache of Google Fonts
+ */
+function llms_local_fonts() {
+	$plugin_url = ( plugins_url( '/lifterlms/', $file ) );
+
+wp_enqueue_style( 'style',  $plugin_url . "assets/css/localstyles.css");
+}
```

Comment:
```suggestion
```

We should drop this... I think we can avoid enqueuing directly through the WP apis for now (knowing that a true webfont API is coming in a future WP core release).

If we drop this we can load them through the existing loader in the dynamic-styles.php (with a few other changes noted there)

However, the `llms_get_certificate_fonts()` function in this file needs to be adjusted to include the CSS file for each font independently.

I'd like to *keep* the google preloading as a fallback through a filter so I'm thinking that if you add a filter to the top of the function, something like:

```php
$use_g_fonts = apply_filters( 'llms_use_google_webfonts', false );
```

Then each `href` prop of the fonts array can be switched to a ternary, eg:

`'href' => $use_g_fonts ? {google-url} : LLMS_PLUGIN . {localurl}`